ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/tree/TomlArrayOfTables.kt

```diff
@@ -65,6 +66,56 @@ public class TomlArrayOfTables(
             (0..index).joinToString(".") { sectionsList[it] }
         }
     }
+
+    override fun TomlEmitter.writeHeader(headerKey: TomlKey, config: TomlConfig) {
```

Comment:
I am starting to think about splitting this config to separate configs: TomlInputConfig and TomlOutputConfig, what do you think about that?

Reply:
Sure, that'd be useful

Reply:
I will create an issue on that https://github.com/akuleshov7/ktoml/issues/135